config/settings.yml

```diff
@@ -674,8 +674,10 @@ claims_api:
   poa_v2:
     disable_jobs: false
   vanotify:
-    representative_template_id: ~
-    service_organization_template_id: ~
+    accepted_representative_template_id: ~
+    accepted_service_organization_template_id: ~
+    declined_representative_template_id: ~
+    declined_service_organization_template_id: ~
```

Comment:
Do we need to update these values in prod/AWS?

Reply:
Yes and we will need to do a manifest update for the change in the name on those as well as the addition of the new ones.

Here is the [previous PR](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/3114)  for the accepted values if that helps at all.  the Production manifest update will have to wait I believe until all the templates get approved so only will need to worry about the lower env additions/updates, but will want to create, or have a ticket created, for the prod ENV additions (there should be one in there already for that so same thing